src/padplus-manager/api-request/room.ts

```diff
@@ -215,4 +216,39 @@ export class PadplusRoom {
     })
   }
 
+  public async acceptRoomInvitation (inviteUrl: string, inviteFrom: string): Promise<void> {
+    log.verbose(PRE, `acceptRoomInvitation(${inviteUrl}, ${inviteFrom})`)
+    const data = {
+      inviteFrom,
+      inviteUrl,
+      type: 'GET_INVITE_INFO',
+    }
+
+    const result = await this.requestClient.request({
+      apiType: ApiType.ACCEPT_ROOM_INVITATION,
+      data,
+    })
+
+    if (result) {
+      const roomDataStr = result.getData()
+
+      if (roomDataStr) {
+        const grpcAccpetRoomInvitation: GrpcAccpetRoomInvitation = JSON.parse(roomDataStr)
+        if (grpcAccpetRoomInvitation && grpcAccpetRoomInvitation.inviteDetailUrl) {
+          try {
+            await axios.post(grpcAccpetRoomInvitation.inviteDetailUrl)
```

Comment:
Let's try to get the response of the post request, if this action failed, I believe that we will receive some meaningful errors.